dbx/about-compatibility.rst

```diff
@@ -1,50 +1,44 @@
+==========================
+About Driver Compatibility
+==========================
 
-We include compatibility tables for each version of the driver to guide
-your decisions on what versions you need to ensure your environment
-remains fully operational.
+Each MongoDB Driver includes two compatibility tables: one for MongoDB and one for the language the driver supports. Before performing an upgrade, use these tables to ensure your environment will remain operational. 
 
-We maintain the following tables for each driver:
-
-- :ref:`About MongoDB compatibility <mongodb-compatibility-table-about-{+driver+}>`
-- :ref:`About language compatibility <language-compatibility-table-about-{+driver+}>`
-
-Read the sections below for detailed explanations of each table.
+Read the following section for detailed explanations of each table.
 
 .. _mongodb-compatibility-table-about-{+driver+}:
 
-About MongoDB Compatibility
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+MongoDB Compatibility Table
+---------------------------
 
-In the **MongoDB Compatibility** table, the columns are labeled with
-versions of MongoDB server and the rows are labeled with major release
-versions of the driver.
+In the **MongoDB Compatibility** table, each column represents a version of 
+version of MongoDB server and each row represents a major release
```

Comment:
issue: I think version here is redundant.

Reply:
great catch.